www/components/social-icons/social-icons.js

```diff
@@ -0,0 +1,24 @@
+import { html, LitElement } from 'lit-element';
+import css from './social-icons.css';
+import githubIcon from '../icons/github-icon';
+import twitterIcon from '../icons/twitter-icon';
+import slackIcon from '../icons/slack-icon';
+
+class SocialIcons extends LitElement {
+  render() {
+    return html`
+    <style>
+      ${css}
+    </style>
+        <a class="icons" href="https://github.com/ProjectEvergreen" target="_blank" rel=”noreferrer noopener” aria-label="open github page">${githubIcon}</a>
+        <a href="https://join.slack.com/t/thegreenhouseio/shared_invite/enQtMzcyMzE2Mjk1MjgwLTU5YmM1MDJiMTg0ODk4MjA4NzUwNWFmZmMxNDY5MTcwM2I0MjYxN2VhOTEwNDU2YWQwOWQzZmY1YzY4MWRlOGI"
+         class="icons slack-icon"
+         target="_blank"
+         rel=”noreferrer noopener”
+         aria-label="slack">${slackIcon}</a>
+        <a href="https://twitter.com/PrjEvergreen" target="_blank" rel=”noreferrer noopener” aria-label="open twitter page">${twitterIcon}</a>
+    `;
+  }
+}
+
+customElements.define('eve-socialicons', SocialIcons);
```

Comment:
Maybe call it something like **social-share-bar**, and we can delimit the the tag name as `<eve-social-share-bar>`?